unit-10/homework/index.js

```diff
@@ -0,0 +1,49 @@
+// Exercise-1 In một bảng nhân của một số bất kỳ với các số từ 1 tới 10 và hiển thị kết quả.
+const calcMultipliers = num => {
+  for (let i = 1; i < 11; i++) {
+    console.log (num * i)
+    }
+}
+
+//Exercise-2 Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 In các số chẵn từ 1 đến n Vd n = 6: in ra: 2 4 6
+const getEvenNumber = num => {
+  if (1 <= num && num <= 30) return;
+
+  for (let i = 1; i <= num; i++) {
+    if (i % 2 === 0)
+    console.log(i)
```

Comment:
Lệnh con phải tab vào 1 tab. so với lệnh cha
```suggestion
    if (i % 2 === 0) {
      console.log(i)
    }
```